src/browser_action.css

```diff
@@ -13,6 +13,31 @@ body {
   padding: 0;
   margin: 0;
 }
+::-webkit-scrollbar {
```

Comment:
Some comment here about what you're doing to the scrollbar would be helpful.

Reply:
I wasn't sure if I should include comments, because this `.css` file hardly has any.

Reply:
Please feel free to add comments when they make sense, especially as more people start to contribute!